src/bindings/python/src/openvino/torch/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from openvino.utils import _add_openvino_libs_to_search_path
+
+_add_openvino_libs_to_search_path()
```

Comment:
I suppose this line is not required. They are removed here https://github.com/openvinotoolkit/openvino/pull/18761

Reply:
Done. Removed